client/src/services/requestService.js

```diff
@@ -15,6 +15,13 @@ axios.interceptors.request.use(
     error => Promise.reject(error)
 );
 
+axios.interceptors.response.use(
+    res => res,
+    err => {
+        throw new Error(err.response.data.error.message);
```

Comment:
make it safety err.?response?.data?.error?.message || err

and rename `err` to `error` and `res` to `response`

Reply:
What will happen if backend is unavailable? Can we propose way to distinguish  like net::ERR_CONNECTION_REFUSED exception  it is created by Axios from  exception that we throw in backend by hand.